modules/deleting-cluster-autoscaler.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+
+:_content-type: PROCEDURE
+[id="deleting-cluster-autoscaler_{context}"]
+= Disabling the cluster autoscaler
+
+To disable the cluster autoscaler, you delete the corresponding `ClusterAutoscaler` resource.
+
+[NOTE]
+====
+Disabling the cluster autoscaler disables autoscaling on the cluster, even if the cluster has existing machine autoscalers.
+====
+
+.Procedure
+
+. List the `ClusterAutoscaler` resource for the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ClusterAutoscaler
+----
++
+.Example output
+[source,terminal]
+----
+NAME      AGE
+default   42m
+----
+
+. Optional: Create a YAML file backup of the `ClusterAutoscaler` CR by running the following command:
++
+[source,terminal]
+----
+$ oc get ClusterAutoscaler/<cluster_autoscaler_name> \//<1>
+  -o yaml> <cluster_autoscaler_name_backup>.yaml //<2>
+----
+<1> `<cluster_autoscaler_name>` is the name of the CR that you want to delete.
+<2> `<cluster_autoscaler_name_backup>` is the name for the backup of the CR.
+
+. Delete the `ClusterAutoscaler` CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete ClusterAutoscaler/<cluster_autoscaler_name>
```

Comment:
May we can set `<cluster_autoscaler_name>` to `default` directly, as in cluster there can only be one clusterautoscaler and the name can only be default.

Reply:
Yes, I think that makes a lot of sense! Thanks for the suggestion :)